addon/templates/components/frost-password.hbs

```diff
@@ -1,8 +1,19 @@
-{{frost-text id=componentId value=value class=class type=type on-input=(action "onInput") disabled=disabled}}
+<div class='row'>
+  {{frost-text
+    id=componentId
+    value=value
+    class=class
+    type=type
+    on-input=(action "onInput")
+    disabled=disabled}}
+  <div class="warning {{if isCapsOn 'showWarning' 'hideWarning'}}">Caps on</div>
+</div>
 {{#if revealable}}
   {{#if showRevealer}}
-    {{#frost-button class='tertiary small revealer' on-click=(action "toggleReveal")}}
-      {{prompt}}
-    {{/frost-button}}
+  <div class='row'>
+    {{#frost-checkbox on-input=(action "toggleReveal")}}
+      Show password
```

Comment:
Is `password` in the spec?  If so I can discuss that with UX - I believe this should only be Show/Hide